### PR TITLE
 simpsons rule added

### PR DESCRIPTION
**Pull request by manildayma**
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- c#
- 

#### Files Added:
- simpsons_rule.cs
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by manildayma** [comment]
@singh-shreya6  please verify

File: Simpsons_rule/simpons_rule.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    { 
+    float res = 0;	    
+    float h = (ul - ll) / n; 
+    float[] x = new float[10]; 
+    float[] fx = new float[10]; 
+	    
+    // Calculatiion of x and f(x) 
+        for (int i = 0; i <= n; i++) 
+        { 
+        x[i] = ll + i * h;
```
Comment (MastersAbh) [inline Simpsons_rule/simpons_rule.cs:24]: please fix indentation

File: Simpsons_rule/simpons_rule.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        x[i] = ll + i * h; 
+        fx[i] = func(x[i]); 
+        } 
+	    
+    // Result Calculation
+        for (int i = 0; i <= n; i++) 
+        { 
+            if (i == 0 || i == n) 
+	        res += fx[i]; 
+	    else if (i % 2 != 0)
```
Comment (MastersAbh) [inline Simpsons_rule/simpons_rule.cs:33]: indentation fix

File: Simpsons_rule/simpons_rule.cs (hunk 1 above)
Comment (manildayma) [reply Simpsons_rule/simpons_rule.cs:24]: done

File: Simpsons_rule/simpons_rule.cs (hunk 2 above)
Comment (manildayma) [reply Simpsons_rule/simpons_rule.cs:33]: done

File: Simpsons_rule/simpons_rule.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    // Function for approximate integral 
+    static float simpsons_(float ll, float ul, int n) 
+    { 
+    float res = 0;	    
+    float h = (ul - ll) / n; 
+    float[] x = new float[10]; 
+    float[] fx = new float[10]; 
+	    
+    // Calculatiion of x and f(x) 
+        for (int i = 0; i <= n; i++)
```
Comment (MastersAbh) [inline Simpsons_rule/simpons_rule.cs:22]: indentation fix

File: Simpsons_rule/simpons_rule.cs [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+	            res += 2 * fx[i]; 
+	    } 
+    res = res * (h / 3); 
+    return res; 
+    } 
+	
+    // MAIN FUNCTION 
+    public static void Main() 
+    {  
+    string val;
```
Comment (MastersAbh) [inline Simpsons_rule/simpons_rule.cs:45]: fix indentation...4 spaces to right

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this PR

**Pull request closed without merging**